### ISSUE TITLE
Add Annotation Toggle View - Inline List Edit of Glosses

### ISSUE THREAD

**Issue by susanodd** [Signbank/Global-signbank#1159]
Add a view for viewing a selection of queried glosses and toggling per row the Tags, Semantic Field, Word Class per gloss, so to avoid going repetitively into Detail View for each gloss.

**Comment by susanodd** [comment]
Here's a quick prototype. This is the idea, that per gloss row, you can quickly toggle tags on or off, without needing to do it via Detail View.

![toggle-tags-view-0](https://github.com/Signbank/Global-signbank/assets/24272544/9d8a6ba6-8f1f-45c3-a242-3af78bcb4749)

![toggle-tags-view-1](https://github.com/Signbank/Global-signbank/assets/24272544/3e97964f-5312-48ce-88bd-4506c34b0ba0)

![toggle-tags-view-2](https://github.com/Signbank/Global-signbank/assets/24272544/ec47c68f-2ea6-4f5d-96c7-a6f209516bdc)

**Comment by susanodd** [comment]
Toggle View is now live on `signbank-susan`
Under the Analysis menu.

**Comment by susanodd** [comment]
More search functionality is needed. At the moment, you can only restrict the results by searching on tags. (On NGT there are thousands of glosses.)
The idea is to allow simple editing to add meta information to glosses without going into Detail View.

**Comment by susanodd** [comment]
@uklomp this code is useable on `signbank-susan `under the Analysis menu, `Annotation Toggle View`.

I'll revised it so the user can choose Recently Added Signs (rather than the entire dataset).
(The search for is rather limited at the moment, so best to select a small dataset to experiment.)

**Comment by susanodd** [comment]
I'll add a search by `created by` to the search form so the user can easily find their own glosses.

**Comment by susanodd** [comment]
This is on signbank-dev

**Comment by susanodd** [comment]
This is on signbank now.

**Comment by uklomp** [comment]
It would be good if these functions are also available in Dutch. Would you be responsible for the translation, Susan?

**Comment by uklomp** [comment]
Indeed, because of this overlap, we will keep semantic fields, but named entities is not necessary.

**Comment by susanodd** [comment]
> Indeed, because of this overlap, we will keep semantic fields, but named entities is not necessary.
> 
> I mean, as a general function/category we should keep them, but not in this toggle overview.

I made a setting to turn the named entity display off for our signbank. Since it was already fully implemented, others can just set the setting to make it visible again.

I added translations for many many views. We hadn't done any translations since last spring.
You'll see these in all the search forms, etc.

One thing: the Tag model has no translations.
The class Tag and TaggedItem are imported models in a python package.
It would be possible to just make some in Dutch in the admin. But those would be "different" tags, not translations of the same.
The tags as they are can be attached to any kind of object. But we only use them on glosses/morphemes.

For the new kinds of tags I can see how it would be to make them translatable.
